versioneer.py

```diff
@@ -0,0 +1,2109 @@
+
+# Version: 0.21
+
+"""The Versioneer - like a rocketeer, but for versions.
```

Comment:
Oh wow this is far less clever than I expected...

Reply:
hopefully in a good way, haha